packages/eservice-template-process/src/services/validators.ts

```diff
@@ -9,3 +15,15 @@ export function assertRequesterEServiceTemplateCreator(
     throw operationForbidden;
   }
 }
+
+export function assertIsDraftEserviceTemplate(
+  eserviceTemplate: EServiceTemplate
+): void {
+  if (
+    eserviceTemplate.versions.some(
```

Comment:
An e-service template is considered a draft, and hence editable, when all its versions are in DRAFT state.
`some` should become `every`